tools/js-sdk-release-tools/src/utils/generateInputUtils.ts

```diff
@@ -21,7 +21,8 @@ async function isManagementPlaneModularClient(specFolder: string, typespecProjec
     }
 
     const tspConfig = await loadTspConfig(tspFolderFromSpecRoot);
-    if (tspConfig?.options?.['@azure-tools/typespec-ts']?.['isModularLibrary'] !== true) {
+    if ((tspConfig?.options?.['@azure-tools/typespec-ts']?.['is-modular-library'] ||
```

Comment:
`is-modular-library` has higher priority. 
In current code: 
if `is-modular-library` == false, `isModularLibrary == true`, then it indicates "not a modular client". but I think it should be "modular client". 

You only need to check "isModularLibrary" when "is-modular-library" is "undefined"